GameVersion.h

```diff
@@ -31,6 +39,7 @@ class GameVersion: commonItems::convenientParser
 	bool operator<=(const GameVersion& rhs) const;
 	bool operator==(const GameVersion& rhs) const;
 	bool operator!=(const GameVersion& rhs) const;
+	bool smallerish(const GameVersion& rhs) const;
```

Comment:
The function returns true when `rhs` is smallerish than `*this`. After reviewing tests I think it would read more naturally if the function returned true when `*this` was smallerish than `rhs`. Is it just me?

Reply:
Eh, can go either way. This way it's left <= right, so similar to actual <= comparator.

Reply:
So, a simple semantics swap (rename) and I think it's now way more clear, if you peek at the tests. Thankyous.

Reply:
Indeed, much clearer now